grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcDecoratingService.java

```diff
@@ -61,6 +61,9 @@ public void serviceAdded(ServiceConfig cfg) throws Exception {
                                                          .config()
                                                          .dependencyInjector();
         decorated = handlerRegistry.applyDecorators(delegate, dependencyInjector);
+        for (HttpService decorator : decorated.values()) {
```

Comment:
I found a bug where `Service.serviceAdded()` isn't called for the gRPC decorators.
it's a relatively simple fix, I've included the change here instead of creating a separate PR.

Reply:
Thank you for the fix! 🙇